packages/spritesheet/src/SpritesheetLoader.ts

```diff
@@ -38,14 +40,14 @@ export class SpritesheetLoader
 
         const loadOptions = {
             crossOrigin: resource.crossOrigin,
-            metadata: resource.metadata.imageMetadata,
+            metadata: (resource.metadata as any).imageMetadata,
```

Comment:
oh no. We cant override metadata anymore! :(

Reply:
All okay while we use v3 of resource loader
https://github.com/pixijs/pixi.js/blob/dev-spritesheet-ts-r/packages/loaders/package.json#L29

Reply:
Then i exposed fields on ResourceLoader class

https://github.com/pixijs/pixi.js/pull/6385/commits/614337808e4329c12e49524cdb770bc4223539a1#diff-aa234d8720b6d5fbb3e622efebb73118R13-R15